tests/Service/ProgrammesAggregationService/FindDescendantEpisodesTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Tests\BBC\ProgrammesPagesService\Service\ProgrammesAggregationService;
+
+use BBC\ProgrammesPagesService\Domain\Entity\Programme;
+
+/**
+ * @covers BBC\ProgrammesPagesService\Service\ProgrammesAggregationService::findDescendantEpisodes
+ */
+class FindDescendantEpisodesTest extends AbstractProgrammesAggregationTest
+{
+    public function testFindDescendantEpisodes()
+    {
+        $mockProgramme = $this->createMock(Programme::class);
+        $mockProgramme->method('getDbAncestryIds')->willReturn([11, 12]);
+
+        $this->mockRepository->expects($this->once())
+                             ->method('findDescendantsByType')
+                             ->willReturn([['type' => 'episode'], ['type' => 'episode']]);
```

Comment:
You're missing an assert on the arguments that get passed to `findDescendantsByType`.

Also please use the `$this->programmesFromDbData($dbData)` pattern for generating and testing the results like in every other Service test.